apps/ansys_mechanical/readme.md

```diff
@@ -0,0 +1,80 @@
+## Install and run Ansys Mechanical Benchmarks
+
+## Prerequisites
+
+Cluster is built with the desired configuration for networking, storage, compute etc. You can see the tutorial or examples folder in this repo for how to set this up.
+
+Recommended cluster setup
+Start with the simple_hpc_pbs example. Before you do azhpc-build copy the scripts directory from <azurehpc>/apps/ansys_mechanical to your cluster build directory. First, you will need to add the following line above the "tags" section for the headnode.
+
+"data_disks": [2048, 2048],
+    
+ Second, add the following section in the scripts section above the pbsdownload piece (~line 90 in the config.json file). 
+
+{
+    "script": "add_pkg.sh",
+     "tag": "add_pkg",
+     "sudo": true
+},
+
+Finally, add "add_pkg" to the tags section for the compute nodes (~line 50 in the config.json file).Once these changes are made, then when you build the cluster (azhpc-build) it will get the neccessary scripts to install the prerequsites on the compute nodes
+
+Dependencies for binary version:
+
+* v19.*
+
+## Installation
+
+NOTE: Update the path to the fluent installer tar file in $azhpc_dir/apps/fluent/install_fluent.sh
```

Comment:
mechanical?

Reply:
should you just refer to the right tutorial configuration instead of documenting the config update ?

Reply:
Done.